editor.planx.uk/src/pages/Team.tsx

```diff
@@ -286,17 +279,28 @@ const Team: React.FC = () => {
   const { id: teamId, slug } = useStore((state) => state.getTeam());
   const [flows, setFlows] = useState<any[] | null>(null);
   const navigation = useNavigation();
+
   const fetchFlows = useCallback(() => {
     useStore
       .getState()
       .getFlows(teamId)
       .then((res: { flows: any[] }) => {
-        setFlows(res.flows);
+        // Copy the array and sort by most recently edited desc using last associated operation.createdAt, not flow.updatedAt
+        const sortedFlows = res.flows
+          .slice()
+          .sort((a, b) =>
+            b.operations[0]["createdAt"].localeCompare(
+              a.operations[0]["createdAt"],
+            ),
+          );
```

Comment:
Previously, we sorted directly via the GraphQL query like `flows (order_by: { updated_at: desc }) {...}` - but we want to sort via operations, not flows, so now we're doing a custom sort here !

Reply:
nit: possibly good use for ToSorted() to copy and sort array with one method